### PR TITLE
DBZ-305 Rebase build process against Kafka 0.11

### PR DESCRIPTION
**Pull request by jpechane**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -55,15 +55,15 @@
...
         <version.curator>2.11.0</version.curator>
-        <version.zookeeper>3.4.8</version.zookeeper>
+        <version.zookeeper>3.4.10</version.zookeeper>
         <version.jackson>2.8.5</version.jackson>
-        <version.org.slf4j>1.7.21</version.org.slf4j>
+        <version.org.slf4j>1.7.25</version.org.slf4j>
         <version.log4j>1.2.17</version.log4j>
         <!-- check new release version at https://github.com/confluentinc/schema-registry/releases -->
-        <version.confluent.platform>3.2.0</version.confluent.platform>
+        <version.confluent.platform>3.2.2</version.confluent.platform>
```
Comment (gunnarmorling) [inline pom.xml]: As per http://docs.confluent.io/3.2.2/release-notes.html, 3.2.2 uses Kafka 0.10.2.1-cp2. Given this CP version is only used for the dependency to the Avro converter, I reckon it doesn't make any difference, though?

**Comment by gunnarmorling** [comment]
@rhauch I'd vote for getting it out as soon as possible. We don't have to do a 0.5.2 if you think the move to Kafka 0.11 justifies a minor version bump and go to 0.6 straight away. Just from reading the release notes of Kafka 0.11, it seems though doing it in a micro (0.5.2) would be fine. Anything speaking against it from your PoV?

**Comment by rhauch** [comment]
Might be nice to release 0.5.2 on the same Kafka version so people can upgrade easily without worrying about client upgrades, and release 0.6 with the Apache Kafka upgrade shortly thereafter (or even right away).

**Comment by gunnarmorling** [comment]
Ok, that works for me. Still having a bit of a hard time with the version scheme of Kafka and its semantics (compatible or not).

**Comment by gunnarmorling** [comment]
Going to merge, now that master is 0.6.

**Pull request merged**